test/1.4/lib/T_state_signal.dml

```diff
@@ -0,0 +1,51 @@
+/*
+  © 2021 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+device test;
+
+import "utility.dml";
+
+saved bool on_raise_called;
```

Comment:
Technically, we dont want to use saved variables for interacting with the model since the name and representation of saved variable attributes are not guaranteed. I would prefer to declare proper attributes for these

Reply:
Ok, I'm not 100% happy with the fact - but I'll change this and raise the issue somewhere else.

Reply:
@JonatanWaern, I disagree: We may not document how saved variables map on attributes, but *we* know the mapping, and it will never change for simple variables, and *if* we change it, then updating the tests is the least of our problems (customer-side checkpoint compatibility is a bigger issue)